### ISSUE TITLE
Proposal of B.1 sublineage with Spike substitutions N440K and E484K and 69/70 deletions, among others

### ISSUE THREAD

**Issue by ha-ruiz75** [cov-lineages/pango-designation#93]
by Katherine Laiton-Donato, Carlos Franco-Muñoz, Diego A. Álvarez-Díaz, Héctor Alejandro Ruiz-Moreno, José A. Usme-Ciro, Jhonnatan Reales-González, Diego Andrés Prada, Sheryll Corchuelo, Maria T. Herrera-Sepúlveda, Julian Naizaque, Gerardo Santamaría, Jorge Rivera, Paola Rojas, Magdalena Wiesner, Martha Lucia Ospina Martínez, Marcela Mercado-Reyes

### Description
Sub-lineage of: **B.1**
Earliest sequence: **2021-01-08** (hCoV-19/Colombia/SUC-INS-VG-1147/2021|EPI_ISL_1220061)
Most recent sequence: **2021-05-14** (hCoV-19/USA/CA-CDC-FG-034613/2021|EPI_ISL_2269521)
Countries circulating: Colombia (55), USA (54), Germany (5), Mexico (3), Spain (2), England (1), Aruba (1); (GISAID sequences up to 2021-05-25)

The first sequence came from a sample collected on January 8, 2021. It was assigned to the B.1 lineage and presented amino acid substitutions in the Spike protein region (N440K, E484K), and deletions (del:11288:9; del: 21766:6; del: 21994-21997:3; del: 27897:5).
New sequences have been appearing, initially in the northwestern region, now spread to other parts of the country. Sequence data shows introduction and spread in the USA.

Pangolin currently designates most sequences as lineage B.1.526. Some sequences, including the earliest sequence, were assigned to the B.1 lineage. Pangolin had assigned most of these sequences to the B.1.160 or B.1.523 lineages in the previous PangoLEARN version. 

Phylogenetic analysis suggest B.1 is the parental lineage and the sequences group apart from B.1.160, B.1.523 and B.1.526.

### List of characteristic substitutions

> ORF1ab: 
> NSP1_V86F
> NSP2_L501F
> NSP3_K501E
> NSP3_T1004I
> NSP3_P1228S
> NSP5_K90R
> NSP5_P241S
> NSP6_S106del*
> NSP6_G107del*
> NSP6_F108del*
> NSP12_K91R
> NSP12_P323L
> NSP13_T153I
> 
> M:
> I82T
> 
> N:
> M234I
> 
> ORF3a:
> Q57H
> 
> ORF7a:
> I110V
> 
> ORF8:
> K2del
> F3X
> 
> Spike:
> H69del*
> V70del*
> T95I
> Y144del*
> F157L
> D614G
> N440K
> E484K
> D950N
> V1228L
> 
> * \*Not present in all sequences. We found several cases where deletions are masked with Ns.

### List of genomes, collected 2021-05-25, attached.
[B_1+N440K+E484K_genomes.txt](https://github.com/cov-lineages/pango-designation/files/6542653/B_1%2BN440K%2BE484K_genomes.txt)

### Evidence
![arbolvertical](https://user-images.githubusercontent.com/34463016/119573509-fb50c880-bd79-11eb-9b49-d207ce578040.png)

- Access the full phylogenetic tree through this link: [microreact tree: Lineage Proposal B.1+S:N440K+S:E484K](https://microreact.org/project/iq5Wyy4Khzq1ZmfHVHXqLv/c942858c) 

 
### Proposed lineage name
Next available designation within parent B.1 lineage (B.1.X)

**Comment by babarlelephant** [comment]
It would be the 4th lineage having the 3 deletions S:69,S:144 and Nsp6 found in B.1.1.7, B.1.525, B.1.620. Only B.1.1.7 doesn't have E484K.
In one week this (small but representing 10% of Colombia cases, which is facing a wave) lineage walked through B.1.523, B.1.160, B.1.526 and now B.1.37.21 (grouping 3 probably unrelated lineages).

**Comment by AngieHinrichs** [comment]
On the UCSC tree this looks like a descendant of B.1.523:

https://nextstrain.org/fetch/hgwdev.gi.ucsc.edu/~angie/pango-designation-93.json?branchLabel=Spike%20mutations&c=pango_lineage&label=nuc%20mutations:G28975T

B.1.523, B.1.160 and several other lineages have C18877T (nsp14:L280 syn), G25563T (ORF3a:Q57H) and C26725T (M:Y71 syn) in common.  Then B.1.523 splits off with G28975T (N:M234I); then this proposed lineage shares A10323G (nsp5:A269G) with part of B.1.523; then it diverges with many mutations.

@aineniamh and @chrisruis as far as I can tell, this meets the criteria listed at https://pango.network/the-pango-nomenclature-system/statement-of-nomenclature-rules/:

1a: monophyletic
1b: distinguished by >=1 evolutionary event (many mutations)
1c: lots of sequences, I'm sure at least 5 will meet the <5% N/ambiguous requirement
1d: >= 1 internal node (many in UCSC's tree even when disregarding S:440 noise)

2a:
(i) geographically distinct from B.1.523 (lots in Colombia, also introductions & transmission in Mexico, US, Germany)
(iii) looks like it grew from tiny numbers in Jan/Feb to many in March and still more in April (how should we quantify this?)
(vi) multiple mutations of interest/concern (S:E484K, H69del, V70del, Y144del, others seen in VOIs)

I support designating this as a new lineage although perhaps as a descendant of B.1.523.  @ha-ruiz75 does your phylogenetic analysis include sequences from [this branch](https://nextstrain.org/fetch/hgwdev.gi.ucsc.edu/~angie/pango-designation-93.json?branchLabel=Spike%20mutations&c=pango_lineage&label=nuc%20mutations:T5614C,C26060T) of B.1.523 with A10323G?  (scroll to bottom of page, click Download Data link, click Metadata (TSV) to get file with sequence names)

**Comment by ha-ruiz75** [comment]
> @ha-ruiz75 does your phylogenetic analysis include sequences from [this branch](https://nextstrain.org/fetch/hgwdev.gi.ucsc.edu/~angie/pango-designation-93.json?branchLabel=Spike%20mutations&c=pango_lineage&label=nuc%20mutations:T5614C,C26060T) of B.1.523 with A10323G?

Thank you, @AngieHinrichs. This is very interesting. 

B.1.523 has not been detected in Colombia [(Official report and no sequences in GISAID)](http://www.ins.gov.co/Noticias/Paginas/coronavirus-genoma.aspx) so I included very few sequences from this lineage. 

Only three sequences from that branch of B.1.523 had been included in our analysis. I have now included the suggested branch

Sequencing volume is just starting to pick up in Colombia, so parental sequence information could have been missed. 

I have included sequences from that B.1.523 branch in the ML tree below. 
B.1.523 and B.1.160 appear to have a close relation consistent with the shared SNPs.
From this [ML tree](https://microreact.org/project/iq5Wyy4Khzq1ZmfHVHXqLv/c942858c), the first sequence of the proposed lineage coming from Colombia, and no B.1.523 detected in Colombia, I would think it is a sublineage of B.1.
![verticalRevision](https://user-images.githubusercontent.com/34463016/120736046-30af9180-c4b1-11eb-9a9b-68aa68deed33.png)
